src/features/offerv2/components/OfferContent/OfferContent.native.test.tsx

```diff
@@ -741,6 +741,93 @@ describe('<OfferContent />', () => {
       }
     )
   })
+
+  describe('Movie calendar', () => {
+    const offer: OfferResponse = {
+      ...offerResponseSnap,
+      subcategoryId: SubcategoryIdEnum.SEANCE_CINE,
+      stocks: [
+        {
+          beginningDatetime: '2024-02-27T11:10:00Z',
+          features: ['VO'],
+          id: 6091,
+          isBookable: false,
+          isExpired: false,
+          isForbiddenToUnderage: false,
+          isSoldOut: true,
+          price: 570,
+        },
+      ],
+    }
+
+    it('should render <MovieCalendar /> when offer is a movie screening', async () => {
+      renderOfferContent({
+        offer,
+      })
+
+      expect(await screen.findByLabelText('Mardi 27 février')).toBeOnTheScreen()
+    })
+
+    it('should render <MovieCalendar /> without duplicated screening dates', async () => {
+      renderOfferContent({
+        offer: {
+          ...offer,
+          stocks: [
+            ...offer.stocks,
+            {
+              beginningDatetime: '2024-02-27T13:10:00Z',
+              features: ['VO'],
+              id: 6091,
+              isBookable: false,
+              isExpired: false,
+              isForbiddenToUnderage: false,
+              isSoldOut: true,
+              price: 570,
+            },
+          ],
+        },
+      })
+
+      expect(await screen.findByLabelText('Mardi 27 février')).toBeOnTheScreen()
+    })
+
+    it('should render <MovieCalendar /> without expired or forbidden to underage screenings', async () => {
```

Comment:
Je sais que t'es pas fan des répetitions mais j'aurais séparé les deux conditions d'échecs :)

Reply:
J'avoue que je ne suis pas fan du tout 😂 mais bon je peux prendre le retour

Reply:
Ou au moins faire un it.each si vraiment tu veux pas te répeter, mais là le soucis c'est qu'en cas de bug ça ralentie de trouver la source.

Reply:
yes je suis d'accord avec nolwen pas de or dans les titres de test c'est une règle de bonne pratique sur le projet
or = plusieurs test